packages/core/src/__tests__/ceramic-anchor.test.ts

```diff
@@ -228,73 +228,6 @@ describe('Ceramic anchoring', () => {
     await ceramic2.close()
   })
 
-  it('test the same doc anchored twice (different Ceramic instances), first one wins)', async () => {
```

Comment:
It does not make sense anymore.

Reply:
How so? It seems like the idea of what the test is trying to test is still relevant no?  That the first anchored update wins?

Reply:
"the first anchored update wins" - that would be a test for conflict resolution, not for in-memory anchoring

Reply:
> And we have such tests

Do we?  I know we have some targeted unit tests of the conflict resolution code itself, but do we have any tests of that logic end-to-end that operate with a full Ceramic node?

Reply:
The test here has nothing to do with Ceramic node behaviour. Two nodes connected to a single CAS instance. They send two anchor requests for the same stream. The assertion is for CAS to decide that one of them wins. That is, that InMemoryAnchorService conducts correct conflict resolution. Since removal of a Ceramic node from CAS side, that behaviour does not make sense.

Reply:
OOOOH.  Yep, I get it now.  I thought the test was anchoring both branches and testing how Ceramic resolved the conflict.  I agree that testing the behavior of the InMemoryAnchorService here is silly